python/foxglove-sdk/poetry.lock

```diff
@@ -1,4 +1,4 @@
-# This file is automatically @generated by Poetry 2.0.0 and should not be changed by hand.
+# This file is automatically @generated by Poetry 2.1.1 and should not be changed by hand.
```

Comment:
I don't see any issues with this; for now it seems reasonable to keep specifying >=2 and manually upgrade.